### ISSUE TITLE
ERROR:pyinspire:Error retrieving results: HTTP Error 410: Gone

### ISSUE THREAD

**Issue by guadagn0** [DavidMStraub/inspiretools#9]
For some time, O(2 weeks), if I issue the command
`blgtobib draft.blg`
I get

`INFO:inspiretools:Looking up reference 1 of 1`
`ERROR:pyinspire:Error retrieving results: HTTP Error 410: Gone`
`ERROR:inspiretools:There was an error in trying to look up reference XYZ.`
The above error occurs also after I updated inspiretools. I'm running python 3.7.5 on a MacOS 10.15.7.
Any help would be appreciated!

**Comment by DavidMStraub** [comment]
I assume `XYZ` is a valid and existing tex key? If so, it means the extraction worked and only the lookup failed. This is also indicated by the fact that it is a HTTP error. But I just checked that this works for me:

```python
from inspiretools.functions import texkey2bib
print(texkey2bib(["Straub:2018kue"]))
```

So there does not to be a fundamental problem with the API or code. I suggest to just open the URL `https://inspirehep.net/api/literature?q=texkey:XYZ` in your browser and see what happens, since this is all the tool actually does. If the 410 pops up also there, it's a problem with the key I suspect. Maybe the paper was put on the index?

**Comment by guadagn0** [comment]
Sorry to have removed the original texkey, which indeed helps with debugging. The texkey is a valid one, as I just checked:
```html
https://inspirehep.net/api/literature?q=texkey:Choi:2011ys
```

Actually, I also tried the two lines you suggest:
```python
from inspiretools.functions import texkey2bib
print(texkey2bib(["Straub:2018kue"]))
```
and noticed they would work with `python3`, but not with `python` (default). In other words, in my machine `python3` has 'frozen' to version 3.7.1 (for compatibility with other stuff), whereas the default `python` is 3.7.5.

In short, in order to install `inspiretools` I now use
<pre>
<b><del>python3</del></b> <b>python</b> -m pip install inspiretools --user
</pre>
and everything works again.

Thanks! :wave:
